twitter_cleanup/authentication.py

```diff
@@ -7,6 +7,15 @@
 class Authentication:
     """Holds authentication data for further usage in the script"""
 
+    __instance = None
+
+    @staticmethod
+    def getInstance():
```

Comment:
`getInstance` does not follow [PEP-8's naming conventions for methods](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables). May we have `get_instance` instead?